main.go

```diff
@@ -53,6 +53,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	if len(os.Args) == 2 && opts.Version {
+		fmt.Println(Version)
+		os.Exit(0)
+	} else if opts.Version {
+		fmt.Fprintf(os.Stderr, "usage: firectl --version\n")
```

Comment:
I don't think we need this to be an error. If `--version` is given, we should just print the version, regardless of what else is on the commandline. Same as with `--help`

Reply:
Fixed!